libraries/chain/include/eosio/chain/webassembly/common.hpp

```diff
@@ -58,6 +58,8 @@ namespace eosio { namespace chain {
     */
    template<typename T>
    struct array_ptr {
+      using type = T;
+      array_ptr() = default;
```

Comment:
leaves value uninitialized

Reply:
Yep.  I think it can be removed now, as it was only used in an unevaluated context, which I cleaned up on the eos-vm side.